config/ncg-vars.conf

```diff
@@ -0,0 +1,35 @@
+# Set ARGO Monitoring Engine hostname
+# (default: hostname)
+# NAGIOS_SERVER = 
+
+# Set following option if you wish to receive
+# email alerts for Nagios internal checks
+# (default: root@localhost)
+# NAGIOS_ADMIN = root@localhost
+
+# ARGO WEB-API root URL
+# (default: https://api.argo.grnet.gr)
+# WEBAPI_ROOT_URL = https://api.argo.grnet.gr
+
+# ARGO WEB-API token must be provided
+# in order to fetch metric profiles
+# (default: )
+# WEBAPI_TOKEN = MY_WEBAPI_TOKEN
+
+# List of metric profiles used for configuring
+# ARGO Monitoring Engine
+# (default: )
+# PROFILES = MON 
+
+# ARGO POEM URL
+# (default: https://poem.egi.eu)
+# POEM_ROOT_URL = https://poem.egi.eu
```

Comment:
default should be poem.argo.grnet.gr

Reply:
Problem is that poem.argo.grnet.gr is superpoem and I need a tenant POEM URL.

Reply:
Add the raining Tenant .